readthedocs/rtd_tests/tests/test_unresolver.py

```diff
@@ -0,0 +1,85 @@
+from django.test import override_settings
+import django_dynamic_fixture as fixture
+import pytest
+
+from readthedocs.rtd_tests.tests.test_resolver import ResolverBase
+from readthedocs.core.unresolver import unresolve
+from readthedocs.projects.models import Domain
+
+
+@override_settings(
+    PUBLIC_DOMAIN='readthedocs.io',
+    RTD_EXTERNAL_VERSION_DOMAIN='dev.readthedocs.build',
```

Comment:
nit: I think it's better if we use the real production value. It's easier to visualize and compare.

Reply:
This is the dev value, which I think makes sense for testing.